admin/customers.php

```diff
@@ -1276,7 +1276,7 @@ function init() {
                     $zc_address_book_count_list = zen_get_customers_address_book($customer['customers_id']);
                     $zc_address_book_count = $zc_address_book_count_list->RecordCount();
                     ?>
-                <td class="dataTableContent text-right"><?php echo $customer['customers_id'] . ($zc_address_book_count == 1 ? TEXT_INFO_ADDRESS_BOOK_COUNT . $zc_address_book_count : '<a href="' . zen_href_link(FILENAME_CUSTOMERS, 'action=list_addresses' . '&cID=' . $customer['customers_id'] . ($_GET['page'] > 0 ? '&page=' . $_GET['page'] : '')) . '">' . TEXT_INFO_ADDRESS_BOOK_COUNT . $zc_address_book_count . '</a>'); ?></td>
+                <td class="dataTableContent text-right"><?php echo $customer['customers_id'] . ($zc_address_book_count == 1 ? TEXT_INFO_ADDRESS_BOOK_COUNT_SINGLE : sprintf(TEXT_INFO_ADDRESS_BOOK_COUNT, zen_href_link(FILENAME_CUSTOMERS, 'action=list_addresses' . '&cID=' . $customer['customers_id'] . ($_GET['page'] > 0 ? '&page=' . $_GET['page'] : '')), $zc_address_book_count) . '</a>'); ?></td>
```

Comment:
Looks like the final ` . </a>` that is added to all entries could come out of this line as well.  

This is based on considering that `TEXT_INFO_ADDRESS_BOOK_COUNT` contains the full html for the link, `TEXT_INFO_ADDRESS_BOOK_COUNT_SINGLE` doesn't contain any html and the mentioned content would be applied to whichever information is provided.

Reply:
oops. you are right,